alf/environments/suite_socialbot.py

```diff
@@ -38,11 +40,61 @@ def is_available():
     return social_bot is not None
 
 
+@alf.configurable
+def transform_reward(reward, reward_cap=1., positive_reward=True):
+    goal_reward = reward
+    if isinstance(reward, (np.ndarray, list)):
+        goal_reward = reward[0]
+    if positive_reward:
+        goal_reward = goal_reward >= 0
+    goal_reward = goal_reward * reward_cap
+    if isinstance(reward, (np.ndarray, list)):
+        reward[0] = goal_reward
+    else:
+        reward = goal_reward
+    return reward
+
+
+@alf.configurable
+def transform_reward_tensor(reward, reward_cap=1., positive_reward=True):
```

Comment:
Where is this function used? If it's only in your conf file, then it should be put there instead.